src/transformers/models/clvp/modeling_clvp.py

```diff
@@ -312,13 +312,18 @@ def forward(
                 key_states[..., :rotary_emb_dim],
                 key_states[..., rotary_emb_dim:],
             )
+            value_rot, value_pass = (
+                value_states[..., :rotary_emb_dim],
+                value_states[..., rotary_emb_dim:],
+            )
 
             cos, sin = rotary_pos_emb.cos().squeeze(0), rotary_pos_emb.sin().squeeze(0)
-            query_rot, key_rot = apply_rotary_pos_emb(query_rot, key_rot, cos, sin, position_ids)
+            query_rot, key_rot, value_rot = apply_rotary_pos_emb(query_rot, key_rot, value_rot, cos, sin, position_ids)
```

Comment:
For CLVP we apply the rotary embedding to the value states too, [official repo link](https://github.com/neonbjb/tortoise-tts/blob/80f89987a5abda5e2b082618cd74f9c7411141dc/tortoise/models/xtransformers.py#L629). 
Sorry, I missed it.

I have modified the `test_full_model_integration` to include the check for this change.

Reply:
Hi @ArthurZucker, I am sorry for the mistake. I was focused on checking the logits of the Decoder Model (which does not use the rotary embeddings thus the logits were properly validated for that one) and didn't focused much into the CLVP Encoder models.

Reply:
No worries! 🤗